docs/insights.md

```diff
@@ -0,0 +1,42 @@
+---
+#https://www.notion.so/n8n/Frontmatter-432c2b8dff1f43d4b1c8d20075510fe4
+description: Insights
+contentType: howto
+---
+
+# Insights
+
+/// info | Feature availability
+The insights summary banner showing activity from the last 7 days is available to all plans. The Insights dashboard is only available on Pro (with limited date ranges) and Enterprise plans. 
```

Comment:
```suggestion
The insights summary banner displays activity from the last 7 days for all plans. The Insights dashboard is only available on Pro (with limited date ranges) and Enterprise plans. 
```

I'd maybe move this below the bulleted list so that there's some context for what is being discussed.